fem/bilinearform_ext.cpp

```diff
@@ -545,6 +545,8 @@ void EABilinearFormExtension::Assemble()
                                                      ea_data_ext,
                                                      i);
    }
+   MFEM_VERIFY(!dynamic_cast<const NCL2FaceRestriction*>(int_face_restrict_lex),
+               "Element Assembly not yet supported on NCMesh.");
```

Comment:
Not easier to just check e.g. `trial_fes->Conforming()`?

Reply:
done in [a84b74d](https://github.com/mfem/mfem/pull/2840/commits/a84b74d625e17ac6ecc336921d982daf06bf2e9a)